src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -106,6 +106,22 @@ public function getName()
 
     public function setRequirements($requirements)
     {
+        if (isset($requirements['_method'])) {
+            if (0 === count($this->methods)) {
+                $this->methods = explode('|', $requirements['_method']);
```

Comment:
Removing it here is to avoid the duplicated deprecation message. The requirement [will be set](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Routing/Route.php#L308-312) in the `setMethods()` method of the `Route` class, won't it?